.papr.yml

```diff
@@ -4,128 +4,64 @@ branches:
     - try
 
 required: true
-context: f25-sanitizer
+context: f25-primary
 
 container:
-    image: projectatomic/ostree-tester
-
+    image: registry.fedoraproject.org/fedora:25
 packages:
-  - libasan
   - git
-  - coccinelle
 
 env:
     CFLAGS: '-fsanitize=undefined -fsanitize-undefined-trap-on-error -fsanitize=address -O2 -Wp,-D_FORTIFY_SOURCE=2'
     ASAN_OPTIONS: 'detect_leaks=0'  # Right now we're not fully clean, but this gets us use-after-free etc
     # TODO when we're doing leak checks: G_SLICE: "always-malloc"
 
-build:
-    config-opts: >
-      --prefix=/usr
-      --libdir=/usr/lib64
-      --enable-installed-tests=exclusive
-      --enable-gtk-doc
-
-# The g-d-t-r timeout is for test-pull-many.sh; if tweaking this,
-# also be sure to change the other cases below
 tests:
-    - make syntax-check
-    - ./tests/ci-commitmessage-submodules.sh
-    - make check
-    - /bin/sh -c 'gnome-desktop-testing-runner -p 0 --timeout $((10 * 60)) libostree/'
+  - ci/ci-commitmessage-submodules.sh
+  - ci/build-check.sh
+  - make syntax-check
```

Comment:
Unfortunately `syntax-check` requires us to have run `configure` (though glancing at the code, not for good reason).

It's tempting to hook up `syntax-check` to `check`.